sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -25,19 +25,25 @@ Use the client library for Azure Key Vault Certificates in your Node.js applicat
 
 ## Getting started
 
+**Prerequisites**: You must have an [Azure subscription](https://azure.microsoft.com/free/) and a
+[Key Vault resource](https://docs.microsoft.com/en-us/azure/key-vault/quick-create-portal) to use this package.
+If you are using this package in a Node.js application, then use Node.js 6.x or higher.
+
+To quickly create the needed Key Vault resources in Azure and to receive a connection string for them, you can deploy our sample template by clicking:
+
+[![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Fkeyvault-certificates%2Ftests-resources.json)
+
 ### Install the package
 
 Install the Azure Key Vault Certificates client library using npm
 
 `npm install @azure/keyvault-certificates`
 
-**Prerequisites**: You must have an [Azure subscription](https://azure.microsoft.com/free/) and a
-[Key Vault resource](https://docs.microsoft.com/en-us/azure/key-vault/quick-create-portal) to use this package.
-If you are using this package in a Node.js application, then use Node.js 6.x or higher.
+### Install the identity library
```

Comment:
Does the identity section really need a header of its own?

Reply:
I think it could go either in the section above with the installation of the package itself or in its own header, but I opted for this to make it less likely that readers would skip over it if they're skimming to the examples. I guess, though, that anyone who is skimming to the sample code will probably also know that they just haven't installed that package when they get errors from `require`, so I could just remove the section header and I think it still reads fine.

Reply:
Since the other keyvault packages have the same formatting, we'll use this for now